api-report/odsp-driver-definitions.api.md

```diff
@@ -34,6 +34,7 @@ export interface ICacheEntry extends IEntry {
 
 // @public (undocumented)
 export interface ICollabSessionOptions {
+    isolateSocketCache?: boolean;
```

Comment:
A bit weird placement, given "collab" session name. Maybe move it up level? We can regroup later as more options are added over time.

Reply:
yep... will move